testbench/rest_server.py

```diff
@@ -541,6 +541,7 @@ def object_delete(bucket_name, object_name):
 
 @gcs.route("/b/<bucket_name>/o/<path:object_name>")
 @retry_test(method="storage.objects.get")
+@retry_test(method="storage.objects.download")
```

Comment:
Just wanted to verify that you tested this out and it works to have multiple wrappers on a single method.

Reply:
I did some testing and they pass running locally and live. Though I'm now concerned about possible chaining effects. I'll dig a bit deeper and change the patch. Thanks!

Reply:
I've found a better place to move the new download wrapper. When downloads perform, the two wrappers are still chained and will both run. I think the conditional checks in [`handle_retry_test_instruction`](https://github.com/googleapis/storage-testbench/blob/main/testbench/common.py#L699-L703) handle them well enough and therefore it works.

There's a better way to apply the new download wrapper by changing the `download_object_get` method. Though it requires changes in the conformance tests across all languages. Without the changes implemented, all tests would break.